### PR TITLE
docs: add MongoDB Authenticated Charts integration guide

### PR DESCRIPTION
**Pull request by eliobischof**
Closes: #3041

### CODE REVIEW

File: docs/docs/guides/integrations/authenticated-mongodb-charts.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+This integration guide shows how you can embed authenticated MongoDB Charts in your web application using ZITADEL as authentication provider.
+
+## Setup ZITADEL Application
+
+Before you can embed an authenticated chart in your application, you have to do a few configuration steps in ZITADEL Console.
+You will need to provide some information about your app. We recommend creating a new app to start from scratch.
+
+1. Navigate to your [Project](https://console.zitadel.ch/projects)
+1. Add a new application at the top of the page.
```
Comment (peintnermax) [inline docs/docs/guides/integrations/authenticated-mongodb-charts.md:13]:
```suggestion
2. Add a new application at the top of the page.
```

File: docs/docs/guides/integrations/authenticated-mongodb-charts.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+1. Select Web application type and continue.
```
Comment (peintnermax) [inline docs/docs/guides/integrations/authenticated-mongodb-charts.md:14]:
```suggestion
3. Select Web application type and continue.
```

File: docs/docs/guides/integrations/authenticated-mongodb-charts.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+1. Use [Authorization Code](../../apis/openidoauth/grant-types#authorization-code) in combination with [Proof Key for Code Exchange (PKCE)](../../apis/openidoauth/grant-types#proof-key-for-code-exchange).
```
Comment (peintnermax) [inline docs/docs/guides/integrations/authenticated-mongodb-charts.md:15]:
```suggestion
4. Use [Authorization Code](../../apis/openidoauth/grant-types#authorization-code) in combination with [Proof Key for Code Exchange (PKCE)](../../apis/openidoauth/grant-types#proof-key-for-code-exchange).
```

File: docs/docs/guides/integrations/authenticated-mongodb-charts.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+1. Skip the redirect settings and confirm the app creation
```
Comment (peintnermax) [inline docs/docs/guides/integrations/authenticated-mongodb-charts.md:16]:
```suggestion
5. Skip the redirect settings and confirm the app creation
```

File: docs/docs/guides/integrations/authenticated-mongodb-charts.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+1. Copy the client ID, you will need to tell MongoDB Charts about it.
```
Comment (peintnermax) [inline docs/docs/guides/integrations/authenticated-mongodb-charts.md:17]:
```suggestion
6. Copy the client ID, you will need to tell MongoDB Charts about it.
```

File: docs/docs/guides/integrations/authenticated-mongodb-charts.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+1. When you created the app, expand its _OIDC Configuration_ section, change the _Auth Token Type_ to _JWT_ and save the change.
```
Comment (peintnermax) [inline docs/docs/guides/integrations/authenticated-mongodb-charts.md:18]:
```suggestion
7. When you created the app, expand its _OIDC Configuration_ section, change the _Auth Token Type_ to _JWT_ and save the change.
```

**Review by peintnermax — CHANGES_REQUESTED** [review]

File: docs/docs/guides/integrations/authenticated-mongodb-charts.md (hunk 5 above)
Comment (mffap) [reply docs/docs/guides/integrations/authenticated-mongodb-charts.md:17]: ... you will need to configure MongoDB Charts with it.

File: docs/docs/guides/integrations/authenticated-mongodb-charts.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+## Setup Custom JWT Provider for MongoDB Charts
+
+Configure ZITADEL as your _Custom JWT Provider_ following the [MongoDB docs](https://docs.mongodb.com/charts/configure-auth-providers/) .
+
+Configure the following values:
+- Signing Algorithm: RS256
+- Signing Key: JWK or JWKS URL
+- JWKS: https://api.zitadel.ch/oauth/v2/keys
+- Audience: Your apps client ID, you copied when you created the ZITADEL app
```
Comment (mffap) [inline docs/docs/guides/integrations/authenticated-mongodb-charts.md:32]:
```suggestion
- Audience: Your app's client ID which you copied when you created the ZITADEL app
```

**Review by mffap — APPROVED** [review]

**Comment by mffap** [comment]
Nice! Easy to understand

File: docs/docs/guides/integrations/authenticated-mongodb-charts.md (hunk 1 above)
Comment (eliobischof) [reply docs/docs/guides/integrations/authenticated-mongodb-charts.md:13]: in rendered markdown, numbers are incremented automatically, but sorting is easier when writing :-)

**Review by peintnermax — APPROVED** [review]

**Pull request merged**